src/Mvc/src/Microsoft.AspNetCore.Mvc.NewtonsoftJson/AnnotatedProblemDetails.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.AspNetCore.Mvc.Formatters
+{
+    internal class AnnotatedProblemDetails
+    {
+        public AnnotatedProblemDetails() { }
```

Comment:
Add a comment here that the default constructor is required for Newtonsoft.Json deserialization